Makefile

```diff
@@ -2,28 +2,31 @@
 
 .PHONY: default docu README.md test all clean check indent wc
 
+KNOWSHEET_SCRIPTS_DIR := ./KnowSheet/scripts
+KNOWSHEET_SCRIPTS_DIR_FULL_PATH := $(shell "$(KNOWSHEET_SCRIPTS_DIR)/KnowSheetReadlink.sh" "$(KNOWSHEET_SCRIPTS_DIR)" )
```

Comment:
Would `KNSHSCRDIR := $(shell "$(KNSHSCRDIR)/KnowSheetReadlink.sh" "$(KNSHSCRDIR)" )` look better to you? Matches GitHub's 654 pixel width perfectly. :)